modules/opensearch/src/OpenSearchTingObject.php

```diff
@@ -89,15 +137,109 @@ public function setOwnerId($owner_id) {
     $this->ownerId = $owner_id;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getClassification() {
+    // Get the first classification.
+    $classification = $this->firstEntry($this->getRecordEntry('dc:subject', 'dkdcplus:DK5'));
+
+    if (empty($classification)) {
+      return FALSE;
+    }
+
+    // Ignore the shorthand for "Skønlitteratur".
+    return $classification === 'sk' ? '' : $classification;
```

Comment:
Returning empty string and false for empty results is odd but perhaps not the time or place to change here.

Reply:
Agreed :)